### ISSUE TITLE
Tapping an unconscious person on the shoulder have a %chance to wake them up

### ISSUE THREAD

**Issue by wanderrful** [KoffeinFlummi/AGM#1991]
Got the idea from a random comment someone made while playing in an Arma group.  Thought it was a great idea and wanted to pass it along.

**Comment by BryanBoru** [comment]
I can see this being a good concept, I do not think there should be any chance until after, say, 50% of the minimum possible time the player could be unconscious. But after a set period of time, tapping having a small percentage of waking up the player (that possibly increase as time goes on) could be a neat gameplay aspect. This is all said without consideration for the coding that would be involved.

**Comment by wanderrful** [comment]
Agreed that after a certain point, probably when blood count is "too low" (maybe less than 50% as you mentioned) it shouldn't work.
